docs/topics/dokka-get-started.md

```diff
@@ -5,46 +5,72 @@ Below you can find simple instructions to help you get started with Dokka.
 <tabs group="build-script">
 <tab title="Gradle Kotlin DSL" group-key="kotlin">
 
-> These instructions reflect Dokka Gradle plugin v1 configuration and tasks. Starting from Dokka 2.0.0, several configuration options, Gradle tasks, and steps to generate your documentation have been updated, including:
->
-> * [Adjust configuration options](dokka-migration.md#adjust-configuration-options)
-> * [Work with multi-module projects](dokka-migration.md#share-dokka-configuration-across-modules)
-> * [Generate documentation with the updated tasks](dokka-migration.md#generate-documentation-with-the-updated-task)
-> * [Specify an output directory](dokka-migration.md#output-directory)
->
-> For more details and the full list of changes in Dokka Gradle Plugin v2, see the [Migration guide](dokka-migration.md).
+> This guide applies to Dokka Gradle plugin (DGP) v2 mode. The previous DGP v1 mode is no longer supported. 
+> If you're upgrading from v1 to v2 mode, see the [Migration guide](dokka-migration.md).
 >
 {style="note"}
 
-Apply the Gradle plugin for Dokka in the root build script of your project:
+**Apply the Gradle Dokka plugin** 
+
+Apply the Dokka Gradle plugin (DGP) in the root build script of your project:
 
 ```kotlin
 plugins {
     id("org.jetbrains.dokka") version "%dokkaVersion%"
 }
 ```
 
-When documenting [multi-project](https://docs.gradle.org/current/userguide/multi_project_builds.html) builds, you need 
-to apply the Gradle plugin within subprojects as well:
+**Document multi-project builds**
 
-```kotlin
-subprojects {
-    apply(plugin = "org.jetbrains.dokka")
-}
+When documenting [multi-project builds](https://docs.gradle.org/current/userguide/multi_project_builds.html),
+you need to apply the plugin to every subproject you want to document. Share Dokka configuration across subprojects 
+by using one of the following approaches:
+
+* Convention plugin
+* Direct configuration in each subproject if you’re not using convention plugins
+
+For more information about sharing Dokka configuration in multi-project builds, 
+see [Multi-project configuration](dokka-gradle.md#multi-project-configuration).
+
+**Generate documentation**
+
+To generate documentation, run the following Gradle task:
+
+```bash
+./gradlew :dokkaGenerate
 ```
 
-To generate documentation, run the following Gradle tasks:
+This task works for both single and multi-project builds.
+
+Run the `dokkaGenerate` task from the aggregating project by prefixing the task with its project path. For example:
+
+```bash
+./gradlew :dokkaGenerate
+
+// OR
+
+./gradlew :aggregatingProject:dokkaGenerate
+```
 
-* `dokkaHtml` for single-project builds
-* `dokkaHtmlMultiModule` for multi-project builds
+Avoid running `./gradlew dokkaGenerate` without a project path (`:`),
+as Gradle tries to run all `dokkaGenerate` tasks across the entire build, which may trigger unnecessary work.
```

Comment:
I'm not sure I interpret this section correctly.
* The first example has just `dokkaGenerate`, but the second one has both, and I'm not sure what's the difference.
* The second example shows both the simple and a prefixed command, but then we say not to use the simple command.

Should it be structured like "To generate for a single-project build, run dokkaGenerate; to generate for a multi-project build, run dokkaGenerate to trigger such tasks across the entire build or specify a project path to only trigger a specific project"?

Reply:
Yeah, that's exactly it, I missed this tiny difference :) the clarification should work.